Switch to downstream fork of cloudkitty

jovial opened the pull request:
This brings in a few additional features not in the regular wallaby release.

jovial committed:
Switch to downstream fork of cloudkitty

This brings in a few additional features not in the regular wallaby release.

markgoddard commented on etc/kayobe/kolla.yml:
```diff
@@ -181,6 +181,13 @@ kolla_build_blocks:
+    RUN pip install git+https://github.com/stackhpc/cloudkitty.git@stackhpc/wallaby
+  horizon_footer: |
+    # Fixes CloudKitty pricing panel: https://github.com/stackhpc/cloudkitty-dashboard/pull/1
+    RUN pip install git+https://github.com/stackhpc/cloudkitty-dashboard.git@stackhpc/wallaby
```
Could you use the kolla_sources approach rather than footers?

jovial replied:
Oh yeah, missed that thanks.

jovial committed:
Use kolla_sources instead of build blocks footer

jovial commented on etc/kayobe/kolla.yml:
```diff
@@ -104,6 +104,17 @@ kolla_sources:
     location: "{{ kolla_bifrost_source_url }}"
     reference: "{{ kolla_bifrost_source_version }}"
+  cloudkitty-base:
+    # Backport of NOTNUMBOOL: https://review.opendev.org/c/openstack/cloudkitty/+/774084
```
Not sure if I should bother maintaining these comments here as they may get out of sync with the branch. Thoughts?

markgoddard replied:
They could go stale, maybe leave them out.

jovial committed:
Remove comments

These could go stale. At the time of writing we are carrying the following patches:

horizon-plugin-cloudkitty-dashboard:
   - Fixes CloudKitty pricing panel: https://github.com/stackhpc/cloudkitty-dashboard/pull/1

cloudkitty-base:
   - Backport of NOTNUMBOOL: https://review.opendev.org/c/openstack/cloudkitty/+/774084
    - Support customising Prometheus queries: https://review.opendev.org/c/openstack/cloudkitty/+/817275

jovial committed:
Fixup for last commit

The last commit removed more than just the comments.
